src/testUtils/mocks/dataSources.ts

```diff
@@ -1614,6 +1615,24 @@ export const createMockEcdsaSignature = (
   return createMockStringCodec(signature) as MockCodec<EcdsaSignature>;
 };
 
+/**
+ * @hidden
+ */
+export const createMockBTreeSet = <T extends Codec>(
+  items: BTreeSet<T> | unknown[]
+): MockCodec<BTreeSet<T>> => {
+  if (isCodec<BTreeSet>(items)) {
+    return items as MockCodec<BTreeSet<T>>;
+  }
+  const res = createMockCodec(items, !items) as unknown as BTreeSet;
+  res.has = sinon.stub().returns(false);
+  (res as any).size = items.length;
+  items.forEach(i => {
+    (res.has as sinon.SinonStub).withArgs(i).returns(true);
```

Comment:
If you store the stub in a variable before assigning it to the result you can use that and avoid having to use this type assertion